packages/@okta/vuepress-site/docs/guides/oauth-for-okta/scopes/index.md

```diff
@@ -0,0 +1,44 @@
+| `okta.groups.manage`     | Allows the app to manage groups in your Okta organization               | [Groups API](/docs/reference/api/groups/#getting-started-with-the-groups-api)|
+| `okta.groups.read`       | Allows the app to read information about groups and their members in your Okta organization| [Groups API](/docs/reference/api/groups/#getting-started-with-the-groups-api)|
+| `okta.idps.manage`       | Allows the app to create and manage Identity Providers in your Okta organization| [Identity Providers API](/docs/reference/api/idps/#getting-started)|
+| `okta.idps.read`         | Allows the app to read information about Identity Providers in your Okta organization| [Identity Providers API](/docs/reference/api/idps/#getting-started)|
+| `okta.inlineHooks.manage`| Allows the app to create and manage Inline Hooks in your Okta organization.| [Inline Hooks API](/docs/reference/api/inline-hooks/)|
+| `okta.inlineHooks.read` | Allows the app to read information about Inline Hooks in your Okta organization.| [Inline Hooks API](/docs/reference/api/inline-hooks/)|
+| `okta.logs.read`         | Allows the app to read information about System Log entries in your Okta organization| [System Log API](/docs/reference/api/system-log/)|
+| `okta.schemas.manage`    | Allows the app to create and manage Schemas in your Okta organization   | [Schemas API](/docs/reference/api/schemas/#getting-started)|
+| `okta.schemas.read`      | Allows the app to read information about Schemas in your Okta organization| [Schemas API](/docs/reference/api/schemas/#getting-started)|
+| `okta.users.manage`      | Allows the app to create new users and read all profile and credential information for users| [Users API](https://developer.okta.com/docs/reference/api/users/#user-operations), [User Lifecycle Operations](/docs/reference/api/users/#lifecycle-operations) <br> The following operations are not OAuth-enabled: [User Consent Grant Operations](/docs/reference/api/users/#user-consent-grant-operations), [Clear User Sessions](/docs/reference/api/users/#clear-user-sessions)|
+| `okta.users.read`        | Allows the app to read any user's profile and credential information      | [Users API](https://developer.okta.com/docs/reference/api/users/#user-operations), [User Lifecycle Operations](/docs/reference/api/users/#lifecycle-operations) <br> The following operations are not OAuth-enabled: [User Consent Grant Operations](/docs/reference/api/users/#user-consent-grant-operations), [Clear User Sessions](/docs/reference/api/users/#clear-user-sessions)|
+| `okta.users.manage.self` | Allows the app to manage the currently signed-in user's profile           |   |
+| `okta.users.read.self`   | Allows the app to read the currently signed-in user's profile and credential information|  |
```

Comment:
Get current user API: https://developer.okta.com/docs/reference/api/users/#get-current-user

Reply:
Link it to this API